docs/sources/administration/change-home-dashboard.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Change the default home dashboard"
+description = "How to replace the default home dashboard"
+keywords = ["grafana", "configuration", "documentation", "home"]
+type = "docs"
+[menu.docs]
+name = "Home Dashboard"
+identifier = "change-home-dashboard"
+parent = "admin"
+weight = 100
++++
+
+# Change the default home dashboard
+
+You can change the default homepage of Grafana in two different ways: change Grafana preferences or change the Grafana configuration.
+
+## Change home dashboard preference
+
+You can choose any of your existing Grafana dashboards to replace the default home dashboard in the preferences.
+
+Only administrators can change organization preferences and they affect all the users in the organization. Changing your own user preferences only affects you.
+User preferences override organization preferences. 
+
+1. Go to the dashboard you want to set as the home dashboard.
+1. Star this dashboard by clicking on the star next to the dashboard title. 
+1. *[Organization preferences]* In **Configuration > Preferences**, select your starred dashboard in the **Home Dashboard** field. 
+1. *[User preferences]* Hover your avatar in the bottom left corner of Grafana and click on **Preferences**. Select your starred dashboard in the **Home Dashboard** field. 
```

Comment:
For the first procedure, use the existing steps 1 and 2, then separate this step into two steps. I recommend adjusting the wording of the second sentence so it is consistent with the previous procedure step 4.